src/drc/api/serializers.py

```diff
@@ -73,11 +82,34 @@ class Meta:
         gegevensgroep = 'ondertekening'
 
 
+class EnkelvoudigInformatieObjectHyperlinkedRelatedField(serializers.HyperlinkedRelatedField):
+    """
+    Custom field to construct the url for models that have a ForeignKey to
+    `EnkelvoudigInformatieObject`
+
+    Needed because the canonical `EnkelvoudigInformatieObjectCanonical` no longer stores
+    the uuid, but the `EnkelvoudigInformatieObject`s related to it do
+    store the uuid
+    """
+    def get_url(self, obj, view_name, view_args, view_kwargs):
+        obj_latest_version = obj.latest_version
+        return super().get_url(obj_latest_version, view_name, view_args, view_kwargs)
+
+    def get_object(self, view_name, view_args, view_kwargs):
+        lookup_value = view_kwargs[self.lookup_url_kwarg]
+        lookup_kwargs = {self.lookup_field: lookup_value}
+        return self.get_queryset().filter(**lookup_kwargs).order_by('-versie').first().canonical
```

Comment:
`.first()` may return `None`, which will result in a `TypeError` if it does. Take a look at the super class on how to handle this properly